modules/stack/variables.tf

```diff
@@ -178,3 +178,9 @@ variable "stack_destructor_enabled" {
   description = "Flag to enable/disable the stack destructor to destroy the resources of the stack before deleting the stack itself"
   default     = false
 }
+
+variable "administrative" {
+  type        = bool
+  description = "Whether this stack can manage others"
```

Comment:
```suggestion
  description = "Whether this stack can manage other stacks"
```

Reply:
This was a copy from the spacelift stack resource but it doesn't mean we have to be consistent with poor wording 😉